demisto_sdk/commands/lint/templates/dockerfile.jinja2

```diff
@@ -13,7 +13,9 @@ RUN chmod -R 775 /devwork
 {# Install requirments and missing deps #}
 RUN printf "{{ pypi_packs | join('\\n') }}" > /devwork/test-requirements.txt
 RUN OS_RELEASE=$(cat /etc/os-release); if echo "$OS_RELEASE" | grep -q "alpine"; \
-then apk add --no-cache --virtual .build-deps python3-dev gcc build-base; fi; \
+then apk add --no-cache --virtual .build-deps python3-dev gcc build-base; \
+elif echo "$OS_RELEASE" | grep -qi "Debian"; \
+then apt-get update && apt-get install -y --no-install-recommends gcc python3-dev;fi; \
 pip install --no-cache-dir -r /devwork/test-requirements.txt; \
 if echo "$OS_RELEASE" | grep -q "alpine"; then apk del .build-deps; fi;
```

Comment:
We should uninstall gcc, python3-dev in Debian as we do with apline

Reply:
> We should uninstall gcc, python3-dev in Debian as we do with apline

yes you are right